app/scripts/modules/utils/EntriesLog.js

```diff
@@ -0,0 +1,203 @@
+const ODataNode = require('./ODataNode.js');
+const multipartmixed2har = require('./multipartmixed2har.js');
+const formatXML = require('prettify-xml');
+
+/**
+ * @constructor
+ */
+function EntriesLog () {
+    this.oEditorContent = {};
+    this.oNoResponseMessage = {};
+    this._index = 0;
+}
+
+/**
+ * Creates Log entry node.
+ * @param {Object} entry - OData entry
+ * @returns {Object} HTML Node
+ */
+EntriesLog.prototype.getEntryNode = function (entry) {
+    let oNode;
+    let aNodes = [];
+    const that = this;
+
+    /**
+     * Finds current OData version.
+     * @param {Object} el - headers element
+     */
+    const odataVersion = entry.response.headers.find(el => el.name.toLowerCase() === 'odata-version' || el.name.toLowerCase() === 'dataserviceversion');
+
+    if (odataVersion && (
+        odataVersion.value === '4.0' ||
```

Comment:
Version 4.01 is already available although not yet supported by UI5.

Reply:
Good to know, thanks! I guess we can make an update here in the nearest future.